Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Dockerfile for setting up reviewdog and ktlint
+
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install --no-install-recommends -y curl \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/davidmerrick/AdventOfCode2019/pullRequest?prid=4961687)